README.md

```diff
@@ -113,3 +115,17 @@ export default {
 
 </script>
 ```
+
+## refreshToken
+
+This will shortcut getting a refresh token from Google, this should be placed in your App.vue on the created page and run on a timer of 45min
+```html
+<script>
+  name: 'App'
+
+  created () {
+  // NOTE: this throws errors when the user is signed out....
```

Comment:
Is there a way to catch those errors?

Reply:
Not sure....maybe just wrap it in a try catch would be the simplest way. My thought was that the user of the plugin could do that bit as it’s not directly related to the plugin.